lib/elixir/test/elixir/port_test.exs

```diff
@@ -18,4 +18,17 @@ defmodule PortTest do
     assert Port.info(port, :registered_name) == nil
     assert Port.info(port) == nil
   end
+
+  test "monitoring functions are inlined by the compiler" do
+    assert expand(quote(do: Port.monitor(port())), __ENV__) ==
+           quote(do: :erlang.monitor(:port, port()))
+
+    assert expand(quote(do: Port.demonitor(port())), __ENV__) ==
+           quote(do: :erlang.demonitor(port()))
+  end
```

Comment:
@josevalim Is this enough testing for you?

Reply:
I don't want to test they are inlined, that's an implementation detail. We should check they actually work. :)
You can start a port that calls `echo` with `foo` as argument.

Reply:
OK, will make it!